app/assets/stylesheets/components/common/_news_signup_sticky.scss

```diff
@@ -0,0 +1,131 @@
+// Newsletter signup sticky footer component
+//
+// Styleguide Newsletter Signup Sticky
+
+.news-signup-sticky {
+  @extend %stripe-bg;
+  @include clearfix;
+  background-color: $color-off-white;
+  margin: $baseline-unit*2 0 0 0;
+  text-align: center;
+  position: relative;
+}
+
+.news-signup-sticky--positioned {
+  visibility: hidden;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  z-index: 100;
```

Comment:
True - is this a 'thing' on the frontend codebase? I remember the blog had z-indexes set here: https://github.com/moneyadviceservice/publify/blob/5531b58545705e971a5b9a9c3bc765c293ecf3cc/app/assets/stylesheets/_settings.scss#L75